plugins/textpattern.md

```diff
@@ -152,6 +151,7 @@ tinymce.init({
     {start: '(c)', replacement: '©'},
     {start: '//brb', replacement: 'Be Right Back'},
     {start: '//heading', replacement: '<h1 style="color: blue">Heading here</h1> <h2>Author: Name here</h2> <p><em>Date: 01/01/2000</em></p> <hr />'},
+    {start: 'div>p*3', replacement: '<div> <p>First</p> <p>Second</p> <p>Third</p> </div>'}, // Emmet like syntax
```

Comment:
Emmet isn't a super common syntax, so this likely needs a link to the emmet docs, either that or we just shouldn't include this.  Reason being uncommon markups/names are going to confuse users.

Reply:
Removed it